.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
The endpoint url is not private information and should be added to the repository. You can keep .env in a .gitignore but update the readme so new developers know that they need to create the file and what they need to put in it.

Reply:
I've added .env.sample file and updated readme to advise possible env variables used in the project.